pages/index.vue

```diff
@@ -15,27 +15,34 @@
           />
         </nuxt-link>
         <DButton
-            name="Go to Events Calendar"
-            variant="dark"
-            size="medium"
-            v-scroll-to="'#scroll'"
-          />
+          v-scroll-to="'#events'"
```

Comment:
why is the double quoting needed? is it because the `v-scroll-to` value is inherently an expression?

Reply:
i guess so? i accept what the examples show me mary :p